metron-platform/metron-enrichment/src/test/java/org/apache/metron/enrichment/integration/EnrichmentIntegrationTest.java

```diff
@@ -153,9 +168,17 @@ public void test() throws Exception {
       fluxComponent.submitTopology();
 
       kafkaComponent.writeMessages(Constants.ENRICHMENT_TOPIC, inputMessages);
-      ProcessorResult<List<Map<String, Object>>> result = runner.process(getProcessor(inputMessages));
+      ProcessorResult<List<Map<String, Object>>> result = runner.process(getProcessor());
       // We expect failures, so we don't care if result returned failure or not
       List<Map<String, Object>> docs = result.getResult();
+      // OTTO>>>>>>>>>>>>>>>>>>>
```

Comment:
*might* not want the OTTO there, unless you want to be immortalized in Metron code @ottobackwards  ;)  Lord knows the `StellarTest` has enough mentions of my name, though, so I can't talk.